packages/main/test/specs/ComboBox.spec.js

```diff
@@ -505,3 +505,110 @@ describe("Accessibility", () => {
 		assert.ok(combo.getProperty("focused"), "ComboBox to be focused");
 	});
 });
+
+describe("Keyboard navigation", () => {
+	it ("Should focus the first item on arrow down and then the input on arrow up", () => {
+		browser.url(`http://localhost:${PORT}/test-resources/pages/ComboBox.html`);
+
+		const combo = $("#combo-grouping");
+		const input = combo.shadow$("#ui5-combobox-input");
+		const arrow = combo.shadow$("[input-icon]");
+		const staticAreaItemClassName = browser.getStaticAreaItemClassName("#combo-grouping");
+		const popover = browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+		let groupItem, listItem;
```

Comment:
Here, it is good to test the value as well. When you are over a group header and when you are over a suggestion

Reply:
I added tests for that previously - when implementing the grouping functionality - the tests are in the "Grouping" module.